web/src/components/GenericItemCard/GenericItemCard.tsx

```diff
@@ -59,14 +64,22 @@ const GenericItemCard: React.FC & GenericItemCardComposition = ({ children }) =>
   );
 };
 
-const GenericItemCardHeading: React.FC<TextProps> = ({ children, ...rest }) => {
+const GenericItemCardHeader: React.FC = ({ children }) => {
   return (
-    <Flex justify="space-between" fontWeight="medium" width={0.7} wordBreak="break-word" {...rest}>
+    <Flex as="header" align="start" mb={2}>
```

Comment:
`start` is not a valid value I believe

```suggestion
    <Flex as="header" align="flex-start" mb={2}>
```

Reply:
Done!